extend/common-interface/config-file.md

```diff
@@ -1,628 +1,824 @@
+                    ]
+                }
+            ]
+        }
+    }
+}
+{% endhighlight %}
+
+#### Input mapping --- filtered table
+
+{% highlight json %}
+{
+    "storage": {
+        "input": {
+            "tables": [
+                {
+                    "source": "in.c-ex-salesforce.Leads",
+                    "destination": "closed_leads.csv",
+                    "where_column": "Status",
+                    "where_values": ["Closed Won", "Closed Lost"],
+                    "where_operator": "eq"
+                }
+            ]
+        }
+    }
+}
+{% endhighlight %}
+
+#### Output mapping --- basic
+Upload `/data/out/tables/out.c-main.data.csv` to `out.c-main.data`.
+
+{% highlight json %}
+{
+    "storage": {
+        "output": {
+            "tables": [
+                {
+                    "source": "out.c-main.data.csv",
+                    "destination": "out.c-main.data"
+                }
+            ]
+        }
+    }
+}
+{% endhighlight %}
+
+#### Output mapping --- headless CSV
+Upload `/data/out/tables/data.csv`, a CSV file without headers on its first line, to the table `out.c-main.data`.
+
+{% highlight json %}
+{
+    "storage": {
+        "output": {
+            "tables": [
+                {
+                    "source": "data.csv",
+                    "destination": "out.c-main.data",
+                    "columns": ["column1", "column2"]
+                }
+            ]
+        }
+    }
+}
+{% endhighlight %}
+
+#### Output mapping --- set additional properties
+Incrementally upload `/data/out/tables/data.csv` to `out.c-main.data`
+with a compound primary key set on the columns `column1` and `column2`.
+
+{% highlight json %}
+{
+    "storage": {
+        "output": {
+            "tables": [
+                {
+                    "source": "data.csv",
+                    "destination": "out.c-main.data",
+                    "incremental": true,
+                    "primary_key": ["column1", "column2"]
+                }
+            ]
+        }
+    }
+}
+{% endhighlight %}
+
+#### Output mapping --- write even if the job fails
+If you have a table that you are updating during the execution of the job 
+and you want to output that table even if the job fails then you can use the `write_always` flag 
+
+{% highlight json %}
+{
+    "storage": {
+        "output": {
+            "tables": [
+                {
+                    "source": "always-output.csv",
+                    "destination": "out.c-main.always-output",
+                    "write_always": true
+                }
+            ]
+        }
+    }
+}
+{% endhighlight %}
+
+#### Output mapping --- delete rows
+Delete data from the `destination` table before uploading the CSV file (only makes sense with `incremental: true`).
+
+The `delete_where` parameter provides a flexible way to specify which records should be deleted before loading new data. It supports time-based filters and multiple filter conditions:
```

Comment:
deleted from the target table before loading new data into it

Reply:
upraveno, přidal jsem to k poslednímu commitu